content/rule_book/material_and_questions.md

```diff
@@ -19,215 +42,234 @@
-- Context is defined as the verse or verses that are required.
-- The quizzer may not continue any non-required verse or verses.
+The question text should be in the form:
 
-##### Quote-these-Two-Verses (Q2V)
+> Quote [book name], chapter [chapter number], verse [verse number].
 
-- Two consecutive verses
-- Used when a verse is not strong enough on its own and needs another verse to explain it or can be used when two verses combined, form an even stronger thought.
+##### Quote 2 Verses (Q2V)
+
+Q2V is used when a verse is not strong enough on its own and needs another verse to explain it or can be used when 2 verses combined to form an even stronger thought.
+
+- Most be 2 consecutive verses
+- The entire 2-verse reference range is part of the question and therefore required in the response if not fully spoken by the quizmaster
+
+The question text should be in the form:
+
+> Quote [book name], chapter [chapter number], verses [verse numbers].
 
 ##### Finish
 
-- The first five words must be unique to all Finish questions of the exact same type (e.g. FTV compared to all FTVs, FT to all FTs, F2V to all F2Vs, FTN to all FTNs).
-- The quizmaster will read the first five words of the question (less if a quizzer jumps)
-- Quizzers may go back and correct their answers if done in the 30-second time limit. The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quizmaster and answer judge that they know where their mistake was, and what the corrected words are.
-- Context is defined as the verse or verses that are required.
-- The quizzer may not continue any non-required verse or verses.
+Finish questions have question text consisting of the 1st 5 words of material starting based on the question type. These 1st 5 words must be unique to all Finish questions of the exact same type; for example, FTV compared to all FTVs, FT to all FTs, F2V to all F2Vs, FTN to all FTNs.
 
-##### Finish-the-Verse (FTV)
+##### Finish the Verse (FTV)
 
-- Begins at the beginning of a single verse and finishes at the end of that verse.
+- Begins at the beginning of a single verse and finishes at the end of that verse
 
-##### Finish-This (FT)
+##### Finish This (FT)
 
-- Begins in the middle of a verse and finishes at the end of the verse. A FT must start at the beginning of a sentence or the beginning of a quotation.
+- Begins in the middle of a verse and finishes at the end of the verse
+- Must start at the beginning of a sentence or the beginning of a quotation
 
-##### Finish-these-Two-Verses (F2V)
+##### Finish 2 Verses (F2V)
```

Comment:
"Finish These 2 Verses (F2V)"

I've always heard "these" included in F2V, but can't remember if "these" is included for Q2V. It should be consistent. I think both should have "these."

Reply:
Unless there's a large cry from the mob to have me burned at the stake, I'm going to say we go with "Quote 2 Verses" and "Finish 2 Verses". It's what the acronyms are derived from. And it's internally consistent.

Reply:
We don't say "Finish Verse" ...

Reply:
True. Doing so would be be broken English. "Quote 2 verses" is not broken English grammar.